### PR TITLE
Cleanup bottom buttons

### PR DESCRIPTION
**Pull request by kenanmamedoff**

### CODE REVIEW

File: brd-ios/breadwallet/src/Redesign/Core/CLEANerSwift/Controllers/VIPTableViewController.swift [hunk 1]
```diff
@@ -87,9 +87,18 @@ class VIPTableViewController<C: CoordinatableRoutes,
         
         tableView.contentInset.top = topInsetValue
         
+        view.addSubview(verticalButtons)
+        verticalButtons.snp.makeConstraints { make in
+            make.leading.bottom.trailing.equalToSuperview()
```
Comment (rcresnik) [inline brd-ios/breadwallet/src/Redesign/Core/CLEANerSwift/Controllers/VIPTableViewController.swift:92]: should there not be a top constraint to tableView.bottom?

File: brd-ios/breadwallet/src/Redesign/Core/CLEANerSwift/Controllers/VIPViewController.swift [hunk 2]
```diff
@@ -40,14 +40,20 @@ class VIPViewController<C: CoordinatableRoutes,
     }()
     
     lazy var infoButton: UIButton = {
```
Comment (rcresnik) [inline brd-ios/breadwallet/src/Redesign/Core/CLEANerSwift/Controllers/VIPViewController.swift:42]: what is this info button? does the base class rly need it? 🤔

File: brd-ios/breadwallet/src/ViewControllers/RecoveryKey/ConfirmRecoveryKeyViewController.swift [hunk 3]
```diff
@@ -19,8 +19,14 @@ class ConfirmRecoveryKeyViewController: BaseRecoveryKeyViewController {
     
     private let firstWordInputView = RecoveryKeyWordInputView()
     private let secondWordInputView = RecoveryKeyWordInputView()
-    private let continueButton = BRDButton(title: L10n.Button.continueAction, type: .primary)
-
+    
+    private lazy var continueButton: FEButton = {
+        let view = FEButton()
+        view.configure(with: Presets.Button.primary)
```
Comment (rcresnik) [inline brd-ios/breadwallet/src/ViewControllers/RecoveryKey/ConfirmRecoveryKeyViewController.swift:25]: make sure 'our' primary button and BRD primary button have the same configs 🙈

**Review by rcresnik — APPROVED** [review]

File: brd-ios/breadwallet/src/Redesign/Core/CLEANerSwift/Controllers/VIPViewController.swift (hunk 2 above)
Comment (kenanmamedoff) [reply brd-ios/breadwallet/src/Redesign/Core/CLEANerSwift/Controllers/VIPViewController.swift:42]: don't know. it is old. can be a part of refactor later.

File: brd-ios/breadwallet/src/Redesign/Core/CLEANerSwift/Controllers/VIPTableViewController.swift (hunk 1 above)
Comment (kenanmamedoff) [reply brd-ios/breadwallet/src/Redesign/Core/CLEANerSwift/Controllers/VIPTableViewController.swift:92]: i rather would add bottom inset as tall as the buttons if needed.

File: brd-ios/breadwallet/src/ViewControllers/RecoveryKey/ConfirmRecoveryKeyViewController.swift (hunk 3 above)
Comment (kenanmamedoff) [reply brd-ios/breadwallet/src/ViewControllers/RecoveryKey/ConfirmRecoveryKeyViewController.swift:25]: loooks ok :D

**Pull request merged**